.github/workflows/gfmrun.yml

```diff
@@ -0,0 +1,49 @@
+name: Test and Build
+
+on:
+  pull_request:
+    branches:
+      master
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  test:
+    name: Test GFMRUN
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v1
+        with:
+          ref: ${{ github.ref }}
+
+      - name: Set up Go 1.13
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.13
```

Comment:
Should we just roll with latest always?

Reply:
I think that could work too

Reply:
I tried with `latest` as the value and removing the `go-version` key altogether but both the times the build failed. I guess we would have to do with this for the moment...